bugbug/repository.py

```diff
@@ -53,7 +54,8 @@ def _transform(commit):
         'deleted': 0,
         'files_modified_num': 0,
         'types': set(),
-        'author_experience': author_experience[commit]
+        'author_experience': author_experience[commit],
+        'author_experience_90_days': author_experience_90_days[commit]
```

Comment:
Put a comma at the end, otherwise the next time we update this we'll have to modify two lines.